src/utils/history.ts

```diff
@@ -21,6 +21,6 @@ type lastLocation = string | null;
 
 export const history = createBrowserHistory();
 
-history.listen((location, action) => {
-  store.dispatch(updateHistoryHelpers(location.pathname));
+history.listen((locat, action) => {
```

Comment:
It is actually correct to use `location` here. Refer to https://github.com/ReactTraining/history#listening